sr_robot_commander/README.md

```diff
@@ -1,2 +1,8 @@
 # sr_robot_commander
-This package contains functionality to send commands to robotic arm 
```

Comment:
Add a top level description of the package (something like this:)

This package contains libraries to easily control the different robots supported by Shadow Robot. They offer access to the basic functionalities of those robots.